bike-erp/src/test/Inventory/Inventory.test.tsx

```diff
@@ -1,8 +1,8 @@
 import Enzyme, { shallow } from "enzyme";
```

Comment:
Could you remove the console.log on this test. I requested it last time, but it was not removed.

Reply:
It has been done 👍 hadn't even noticed them. If I had, I would have removed them 👍